docs/src/pages/components/buttons/UploadButtons.tsx

```diff
@@ -1,50 +1,35 @@
 import * as React from 'react';
-import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import { experimentalStyled as styled } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
 import Button from '@material-ui/core/Button';
 import IconButton from '@material-ui/core/IconButton';
 import PhotoCamera from '@material-ui/icons/PhotoCamera';
 
-const useStyles = makeStyles((theme: Theme) =>
-  createStyles({
-    root: {
-      '& > *': {
-        margin: theme.spacing(1),
-      },
-    },
-    input: {
-      display: 'none',
-    },
-  }),
-);
+const Input = styled('input')`
+  display: none;
+`;
```

Comment:
```suggestion
const Input = styled('input')({
  display: 'none',
});
```

Reply:
Done. Corrections are made to both files (.js and .tsx).

💡 I thought we would use literal string template